integration-test/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitSpecificIntegrationTestBase.java

```diff
@@ -2957,16 +2958,21 @@ void put_WhenTheOtherTransactionAbortDueToConflict_ShouldBeCommittedWithoutBlock
     failedTxn.get(prepareGet(1, 0, namespace1, TABLE_1));
     failedTxn.put(preparePut(0, 0, namespace1, TABLE_1));
     successTxn.put(preparePut(1, 0, namespace1, TABLE_1));
+    CountDownLatch sync = new CountDownLatch(1);
     Future<?> future =
         executorService.submit(
             () -> {
               try {
+                // This transaction will be committed after the other transaction in the same group
+                // is moved to a delayed group.
                 successTxn.commit();
+                sync.countDown();
               } catch (CommitException | UnknownTransactionStatusException e) {
                 throw new RuntimeException(e);
               }
             });
     executorService.shutdown();
+    sync.await();
```

Comment:
Or can we use `executorService.awaitTermination()`?

Reply:
Good point. Yes, I think so.

This might be just a matter of taste, but I think `executorService.awaitTermination()` more focuses on the release timing of all the threads. So, using the countdown latch is a bit clearer since it's just for synchronization. Although I don't have a strong opinion on it.

Reply:
Another option is `future.get()`? But I don't have any strong opinion on it, either. I'm okay to leave it as it is.

Reply:
@brfrn169 Yes! I thought `future.get()` is the best way, so tried to revise the PR. But, I noticed all the test cases for the group commit feature in the integration test class didn't need multi-threads. Then, I revised this PR not to use multi-threads to make them robust. I also added a missing test case.

Sorry for asking you after your approval, but can you take a look at them again when you get a chance?